File: uri/go/1007.go
package main

import (
	"fmt"
)

func main() {

	var A int
	var B int
	var C int
	var D int
	fmt.Scanf("%d\n%d\n%d\n%d", &A, &B, &C, &D)
	diferenca := (A * B) - (C * D)
	
	fmt.Printf("DIFERENCA = %d\n", diferenca)

}

File: uri/haskell/1007.hs
main = do
    a <- getLine
    b <- getLine
    c <- getLine
    d <- getLine
    let n1 = (read a :: Int)
    let n2 = (read b :: Int)
    let n3 = (read c :: Int)
    let n4 = (read d :: Int)
    let diferenca = (n1 * n2) - (n3 * n4)
    putStrLn ("DIFERENCA = " ++ show diferenca)